### PR TITLE
Helm redis

### PR DESCRIPTION
**Pull request by mariusandra**
Here are some changes that would be cool to get into master. These are required to get the helm charts working with redis and workers. Then when we release a new docker image (either `1.0.11.1` or then `1.1.0` on Wednesday), we should merge the PR for the posthog/charts repo: https://github.com/PostHog/charts/pull/1

### CODE REVIEW

**Comment by timgl** [comment]
Tests failing but otherwise happy with this

**Comment by mariusandra** [comment]
image
;)

File: bin/docker-worker [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
   echo "⚠️ Redis will be mandatory in the next versions of PostHog (1.1.0+)."
   echo "⚠️ Please configure it now to avoid future surprises!"
   echo "⚠️"
   echo "⚠️ See here for more information!"
   echo "⚠️ --> https://docs.posthog.com/#/upgrading-posthog?id=upgrading-from-before-1011"
   echo "⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️"
 else
-  celery -A posthog beat -S redbeat.RedBeatScheduler &
-  celery -A posthog worker
+  ./bin/docker-worker-beat &
```
Comment (timgl) [inline bin/docker-worker:17]: I'm not sure about splitting this up into _that_ many files but also not blocking for merging this

**Review by timgl — APPROVED** [review]

File: bin/docker-worker (hunk 1 above)
Comment (mariusandra) [reply bin/docker-worker:17]: Yeah, it's a lot, but in k8s we will run the beat and worker in separate pods... but for users who just do `docker run` and just get it over with, it'll all be in one command. So it kind of makes sense... 🤷‍♂️ 

image

Comment (timgl) [reply bin/docker-worker:17]: That's fine 👍

**Pull request merged**